FBC/FinancialInstruments/InstrumentPricing.rdf

```diff
@@ -0,0 +1,478 @@
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">This method itself changes quite frequently i.e. the exchange may change the way it computes closing prices.</fibo-fnd-utl-av:explanatoryNote>
+	</owl:ObjectProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-fbc-fi-ip;hasPricingSource">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-rel-rel;refersTo"/>
+		<rdfs:label xml:lang="en">has pricing source</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fbc-fi-ip;SecurityPrice"/>
+		<rdfs:range>
+			<owl:Class>
+				<owl:unionOf rdf:parseType="Collection">
+					<rdf:Description rdf:about="&fibo-be-fct-pub;Publisher">
+					</rdf:Description>
+					<rdf:Description rdf:about="&fibo-fbc-pas-fpas;FinancialServiceProvider">
+					</rdf:Description>
+					<rdf:Description rdf:about="&fibo-fbc-fi-ip;PricingModel">
+					</rdf:Description>
+					<rdf:Description rdf:about="&fibo-fbc-fct-mkt;Exchange">
+					</rdf:Description>
+					<rdf:Description rdf:about="&fibo-fbc-fi-ip;CompositeMarket">
+					</rdf:Description>
+				</owl:unionOf>
+			</owl:Class>
+		</rdfs:range>
+		<skos:definition xml:lang="en">indicates the origin of a given price for a financial instrument</skos:definition>
+	</owl:ObjectProperty>
+	
+	<owl:DatatypeProperty rdf:about="&fibo-fbc-fi-ip;hasQuoteLotSize">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-qt-qtu;hasNumericValue"/>
+		<rdfs:label xml:lang="en">has quote lot size</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fbc-fi-ip;SecurityPrice"/>
+		<rdfs:range rdf:resource="&xsd;positiveInteger"/>
+		<skos:definition xml:lang="en">magnitude of the offering, order or trade to which the quote price refers (i.e., number of shares)</skos:definition>
+	</owl:DatatypeProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-fbc-fi-ip;hasRateOfReturn">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-qt-qtu;hasQuantityValue"/>
+		<rdfs:label xml:lang="en">has rate of return</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fbc-fi-fi;FinancialInstrument"/>
+		<rdfs:range rdf:resource="&fibo-fbc-fi-ip;RateOfReturn"/>
+	</owl:ObjectProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-fbc-fi-ip;hasTradingDateTime">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-dt-fd;hasDateTime"/>
+		<rdfs:label xml:lang="en">has trading date time</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fbc-fi-ip;SecurityPrice"/>
+		<rdfs:range rdf:resource="&fibo-fnd-dt-fd;DateTime"/>
+		<skos:definition xml:lang="en">indicates the specific date and time associated with a given price</skos:definition>
+	</owl:ObjectProperty>
+	
+	<owl:DatatypeProperty rdf:about="&fibo-fbc-fi-ip;hasTradingVolume">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-qt-qtu;hasNumericValue"/>
+		<rdfs:label xml:lang="en">has trading volume</rdfs:label>
+		<rdfs:range rdf:resource="&xsd;decimal"/>
+		<skos:definition xml:lang="en">indicates the number of shares/units traded on a given trading day</skos:definition>
```

Comment:
where is the day indicated?

Reply:
One would have to do that using a separate property, unless you want this concept reified as a class...

Reply:
Depends what this property is applied to (there is no domain) - something needs to have the date.

Reply:
I'm with Pete here - the date has to be somewhere.  I would vote for reification, but I can't tell where it should go, since this property has no domain. 

I guess what I would expect is that hasTradingVolume would point to something that has a volume measurement (a decimal) and a period (probably a fibo-fnd-dt-fd:ExplicitDatePeriod), since trading volume is normally  measured by trading day, but can be measured by other intervals.   Does one ever measure trading volume for less than one day?  If so, ExplicitDatePeriod might not be sufficient; do we have a class for periods of less than one day?

Reply:
I think this is a good point, and we should raise a new issue, but I would raise it in securities potentially, not in FBC.  The challenge here would be to address the circularity introduced by referencing the listing on a specific exchange, for example that has some trading volume over some period, which could be a matter of hours.   I'll have to try an example to see iif our concept of duration can be less than a full day, and if not, we would need to fix that in FND.  Then we would need to try to represent some average price over that duration using this ontology as well as the information for the listing, to make sure this works properly.